make rpm fails at "/usr/lib/rpm/rpmuncompress -x /home/achilleas/projects/osbuild/image-builder-cli/rpmbuild/SOURCES/image-builder-cli-44.tar.gz"

```
› make rpm
mkdir -p /home/achilleas/projects/osbuild/image-builder-cli/rpmbuild/SPECS
git show HEAD:image-builder.spec > rpmbuild/SPECS/image-builder.spec
go mod vendor
go: downloading github.com/osbuild/images v0.228.0
go: downloading github.com/osbuild/blueprint v1.18.0
go: downloading github.com/spf13/cobra v1.10.2
go: downloading github.com/spf13/pflag v1.0.10
go: downloading go.yaml.in/yaml/v3 v3.0.4
go: downloading sigs.k8s.io/yaml v1.6.0
go: downloading github.com/IBM/ibm-cos-sdk-go v1.12.3
go: downloading github.com/gophercloud/gophercloud/v2 v2.8.0
go: downloading github.com/IBM/go-sdk-core/v5 v5.21.0
go: downloading github.com/Microsoft/hcsshim v0.13.0
go: downloading github.com/go-playground/validator/v10 v10.26.0
go: downloading go.mongodb.org/mongo-driver v1.17.2
go: downloading github.com/go-playground/universal-translator v0.18.1
go: downloading github.com/gabriel-vasile/mimetype v1.4.8
go: downloading github.com/leodido/go-urn v1.4.0
go: downloading github.com/containerd/cgroups/v3 v3.0.5
go: downloading github.com/go-playground/locales v0.14.1
go: downloading github.com/golang/groupcache v0.0.0-20241129210726-2c02b8208cf8
./tools/rpm_spec_add_provides_bundle.sh rpmbuild/SPECS/image-builder.spec
mkdir -p /home/achilleas/projects/osbuild/image-builder-cli/rpmbuild/SOURCES
git archive --prefix=image-builder-cli-dc16ea1b8797d3e13d85c1536d93a16ac7a0d492/ --format=tar.gz HEAD > rpmbuild/SOURCES/image-builder-cli-dc16ea1b8797d3e13d85c1536d93a16ac7a0d492.tar.gz
gunzip -f rpmbuild/SOURCES/image-builder-cli-dc16ea1b8797d3e13d85c1536d93a16ac7a0d492.tar.gz
tar --delete --owner=0 --group=0 --file rpmbuild/SOURCES/image-builder-cli-dc16ea1b8797d3e13d85c1536d93a16ac7a0d492.tar image-builder-cli-dc16ea1b8797d3e13d85c1536d93a16ac7a0d492/image-builder.spec
tar --append --owner=0 --group=0 --transform "s;^;image-builder-cli-dc16ea1b8797d3e13d85c1536d93a16ac7a0d492/;" --file rpmbuild/SOURCES/image-builder-cli-dc16ea1b8797d3e13d85c1536d93a16ac7a0d492.tar rpmbuild/SPECS/image-builder.spec vendor/
tar --append --owner=0 --group=0 --transform "s;rpmbuild/SPECS/;image-builder-cli-dc16ea1b8797d3e13d85c1536d93a16ac7a0d492/;" --file rpmbuild/SOURCES/image-builder-cli-dc16ea1b8797d3e13d85c1536d93a16ac7a0d492.tar rpmbuild/SPECS/image-builder.spec
gzip rpmbuild/SOURCES/image-builder-cli-dc16ea1b8797d3e13d85c1536d93a16ac7a0d492.tar
rpmbuild -bb \
        --define "_topdir /home/achilleas/projects/osbuild/image-builder-cli/rpmbuild" \
        --define "commit dc16ea1b8797d3e13d85c1536d93a16ac7a0d492" \
        --with tests \
        rpmbuild/SPECS/image-builder.spec
setting SOURCE_DATE_EPOCH=1737676800
Executing(%mkbuilddir): /bin/sh -e /var/tmp/rpm-tmp.KKZxKi
Executing(%prep): /bin/sh -e /var/tmp/rpm-tmp.KQcuHg
+ umask 022
+ cd /home/achilleas/projects/osbuild/image-builder-cli/rpmbuild/BUILD/image-builder-44-build
+ cd /home/achilleas/projects/osbuild/image-builder-cli/rpmbuild/BUILD/image-builder-44-build
+ rm -rf image-builder-cli-dc16ea1b8797d3e13d85c1536d93a16ac7a0d492
+ /usr/lib/rpm/rpmuncompress -x /home/achilleas/projects/osbuild/image-builder-cli/rpmbuild/SOURCES/image-builder-cli-44.tar.gz
error: File /home/achilleas/projects/osbuild/image-builder-cli/rpmbuild/SOURCES/image-builder-cli-44.tar.gz: No such file or directory
error: Bad exit status from /var/tmp/rpm-tmp.KQcuHg (%prep)

RPM build errors:
    Bad exit status from /var/tmp/rpm-tmp.KQcuHg (%prep)
make: *** [Makefile:161: rpm] Error 1
```

The archive is actually created with the commit ID instead of the version tag
```
› ls rpmbuild/SOURCES/
image-builder-cli-dc16ea1b8797d3e13d85c1536d93a16ac7a0d492.tar.gz
```

--- a/image-builder.spec
+++ b/image-builder.spec
@@ -27,7 +27,7 @@ ExcludeArch:    i686
 License:        Apache-2.0 AND BSD-2-Clause AND BSD-3-Clause AND CC-BY-SA-4.0 AND ISC AND MIT AND MPL-2.0 AND Unlicense
 
 URL:            %{gourl}
-Source0:        https://github.com/osbuild/image-builder-cli/releases/download/v%{version}/image-builder-cli-%{version}.tar.gz
+Source0:        %{gosource}
 
 
 BuildRequires:  %{?go_compiler:compiler(go-compiler)}%{!?go_compiler:golang}

